projects/observability/src/pages/api-trace-detail/api-trace-detail.page.component.ts

```diff
@@ -31,12 +33,19 @@ import { ApiTraceDetails, ApiTraceDetailService } from './api-trace-detail.servi
             icon="${IconType.Time}"
             [value]="traceDetails.timeString"
           ></ht-summary-value>
-          <ht-summary-value
-            class="summary-value"
-            icon="${IconType.TraceId}"
-            label="Trace ID"
-            [value]="traceDetails.traceId"
-          ></ht-summary-value>
+
+          <ng-container
+            *ngIf="this.getExploreNavigationParams(traceDetails.traceId) | async as exploreNavigationParams"
+          >
+            <ht-link class="link" [paramsOrUrl]="exploreNavigationParams">
```

Comment:
Could you add a screenshot of what this looks like after the change? Specifically, I'm looking for what this looks like vs what it would look like to use `ExploreFilterLinkComponent` instead of htLink directly.

Reply:
![Screenshot 2021-09-01 at 3 42 12 PM](https://user-images.githubusercontent.com/86998924/131653940-9c9532e7-b840-4005-aeb1-31ed29790674.png)
while hovering we can see cursor pointer and tooltip , otherwise no visual changes

Reply:
https://user-images.githubusercontent.com/86998924/131654660-0d072b70-a784-4b0a-89cd-b6da3f981a24.mov

please check uploaded a small video

Reply:
got it, thanks - I think for consistency, it would make since to use the filter link here. It also gives a visual indicator, without requiring a hover, that it's actionable.

Something like:
```html
<ht-explore-filter-link [paramsOrUrl]="this.getExploreNavigationParams(traceDetails.traceId) | async"
  <ht-summary-value
    class="summary-value"
    icon="${IconType.TraceId}"
    label="Trace ID"
    [value]="traceDetails.traceId">
  </ht-summary-value>
</ht-explore-filter-link>

Reply:
Yep, I thought we recommended `ht-explore-filter-link` only. It is same as ht-link but it brings in an icon at the end that indicates the user what type of link it is.